corehq/apps/reports/standard/deployments.py

```diff
@@ -165,18 +165,20 @@ def rows(self):
 
         user_id = self.request.GET.get('individual')
         if not user_id:
-            return []
-
-        # security check
-        get_document_or_404(CommCareUser, self.domain, user_id)
-
+            key_args = {}
```

Comment:
other options would be:
- use the domain/docs view somehow
- grab all the user IDs in the domain first and then query by those

I don't think either of those will be particularly performant or easy to order though, so happy to just force people to pick.